homeassistant/helpers/singleton.py

```diff
@@ -9,9 +9,9 @@
 from homeassistant.core import HomeAssistant
 from homeassistant.loader import bind_hass
 
-T = TypeVar("T")
+_T = TypeVar("_T")
 
-FUNC = Callable[[HomeAssistant], T]
+FUNC = Callable[[HomeAssistant], _T]
```

Comment:
Same as my comment on part 1.
PEP 484 recommends using CamelCase for Type Aliases.

Reply:
AFAICT `FUNC` isn't imported anywhere else, so this would probably be safe to do. However, I'd like to focus these PRs solely on TypeVars.

I could do a followup PR, but it's not really worth it IMO. If at some point we decide to rename Type Aliases, we can do this one too.